Documentation/networking/overview.md

```diff
@@ -308,6 +308,56 @@ The basic network configuration is as follows:
 This will set up a linux-bridge, connect the container to the bridge and assign container IPs out of the subnet that flannel assigned to the host.
 For more information included advanced configuration options, see [CNI docs][cni-flannel].
 
+## Custom plugins
+
+Apart from the aforementioned plugins bundled with rkt, it is possible to run custom plugins that implement the [CNI (Container Network Interface)][cni].
+CNI plugins are just binaries that receive a json configuration file and rkt looks for plugin binaries and configuration files in certain well-defined locations.
+
+As we saw before, the default location where rkt looks for CNI configurations is `$LOCAL_CONFIG_DIRECTORY/net.d/`, where `$LOCAL_CONFIG_DIRECTORY` is `/etc/rkt` by default (it can be changed with rkt's `--local-config` flag).
```

Comment:
Are you confident that the default path and the option for modifying the config directory will not change? If not I suggest to remove this information here and just reference the source of truth for this information.

Reply:
Since we already mention `/etc/rkt` here and I don't think it's likely to change, I'm comfortable keeping this.